README.md

```diff
@@ -428,7 +428,7 @@ These constructs are not in the scope of this proposal, but could be added by se
 
 [JSX](https://facebook.github.io/jsx/) is an XML-like syntax extension to JavaScript that is designed to be transformed by a pre-processor into valid JavaScript.
 It is widely used in [the React ecosystem](https://reactjs.org/docs/introducing-jsx.html), but it has been used for different libraries and frameworks.
-Because JSX directly interleaves with JavaScript code, compilers and type-checkers for JavaScript typically also support checking and transforming JSX as well.
+Because JSX directly interleaves with JavaScript code, compilers and type-checkers for JavaScript typically while also support checking and transforming JSX as well.
```

Comment:
This one's wrong, I believe. I propose to return it back to the original, which seems fine.

```suggestion
Because JSX directly interleaves with JavaScript code, compilers and type-checkers for JavaScript typically also support checking and transforming JSX as well.
```